karspexet/ticket/models.py

```diff
@@ -36,6 +39,11 @@ def seats(self):
     def total_price(self):
         return reduce((lambda acc, seat: acc + int(seat.price_for_type(self.tickets[str(seat.id)]))), self.seats(), 0)
 
+    def save(self, *args, **kwargs):
```

Comment:
It's kinda dangerous to override save, since there are a number of different ways to create/save objects without calling save (e.g. [bulk_create](https://docs.djangoproject.com/en/1.11/ref/models/querysets/#bulk-create)). Also, this means new Reservation objects have inconsistent state until saved.

Instead just use a callable as the [default](https://docs.djangoproject.com/en/1.11/ref/models/fields/#default) value.